Feature

tasnimulhasan07 opened the pull request:
create front mobile page with html and add containers and items and using css create properties for each item then add images by extracting from figma templates and use linters to ensure correct structure of code

tasnimulhasan07 committed:
create linters

add linters to project to ensure collaborative readability

tasnimulhasan07 committed:
create gitignore file

create gitignore file to store files to be ignored

tasnimulhasan07 committed:
add microverse linters

add linters to correct code

tasnimulhasan07 committed:
hamburger

add hamburger for nav

tasnimulhasan07 committed:
add images

add images for mobile skeleton

tasnimulhasan07 committed:
create index

create html file to create containers and items for nav and front page

tasnimulhasan07 committed:
create styles

create css file to ensure all html items and containers have correct properties relative to template from figma

tasnimulhasan07 committed:
read me

update readme file to explain project

tasnimulhasan07 committed:
add images

add images for front page

tasnimulhasan07 committed:
add linter

add linter

tasnimulhasan07 committed:
update index

add html file for front page with containers and items and nav

tasnimulhasan07 committed:
update css

add css file with properties for each item and container on html file

Amrhub commented on index.html:
```diff
@@ -0,0 +1,79 @@
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+        body{background: url("Image/backimg.png");
```
- [x] Let's make sure we add this background image to the `headline` section instead of the body because if it's the body it will extend as we extend our body content. 
The issue already occur if you scroll a little bit down

Amrhub commented on index.html:
```diff
@@ -0,0 +1,79 @@
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <br>
+            <br>
+            <br>
```
- [x] [OPTIONAL] I would recommend we use CSS to give margins and padding instead of using the `<br>` tag for spacing

Amrhub reviewed (changes requested):
Hi @tasnimulhasan07,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Professional documentation ✔️ 
Highly organized code ✔️ 👏🏻 
Meaningful commit messages 🤓 

## Required Changes ♻️
_Check the comments under the review._
![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tasnimulhasan07 committed:
update html and css

remove background and put into homepage section and remove br and add margin and padding

tasnimulhasan07 commented:
added background to homepage section instead of body and removed br from html and added padding and margin to css

tasnimulhasan07 committed:
css

semi colon

Wes-Isaac commented on styles.css:
```diff
@@ -0,0 +1,137 @@
+  }
+}
+
+.homepage {
+  background: url("Image/backimg.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  height: 718px;
+}
```
- [x] Great job fixing what the previous reviewer requested, however, your page is different from the design on Figma. Kindly fix that.
- [x] Please make sure that you use the same font, spacing, color specified in the design.
Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154070487-d80badd7-107e-4efe-80c0-d46a117af966.png) | ![image](https://user-images.githubusercontent.com/59416136/154070707-18c5f3fd-ed17-40a8-87b7-4dce79d53ca3.png)

tasnimulhasan07 replied:
fixed background not matching from desktop image to mobile image, fixed alignment issue, updated linked in image and fixed index and styles codes for it.

Wes-Isaac reviewed (changes requested):
Hi @tasnimulhasan07 

Impressive work so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!

### 📝👌 Your project highlights 🔥😎

- Linters are on point ✔
- Splendid work with the README file 👀
- Proper use of GitHub flow 👌
- Awesome design 🙌

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tasnimulhasan07 committed:
updated images

updated background and linked in image

tasnimulhasan07 committed:
update index

added missing linked in image

tasnimulhasan07 committed:
update css

fixed background issue and alignment issue

keroloussamy commented:
@tasnimulhasan07 Where are the changes?

tasnimulhasan07 commented:
Sorry I forgot to push from gitdesk top give me 2 mins

On Thu, 17 Feb 2022, 2:41 am Kerolous Samy Grace, ***@***.***>
wrote:

> @tasnimulhasan07 <https://github.com/tasnimulhasan07> Where are the
> changes?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tasnimulhasan07/skel-4/pull/1#issuecomment-1042511136>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AU5UHQX5LCEODYJQPBISA53U3RN4VANCNFSM5OOFJAUQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mateo951 commented:
Don't worry take your time 🤓

tasnimulhasan07 committed:
fix linter errors

fix linter errors

tasnimulhasan07 commented:
all issues fixed and pushed, can someone finish the review now, please

tasnimulhasan07 commented:
Hi, just leaving double MSG's, to be sure someone sees this, I have fixed
all linter errors and pushed final piece, can someone review the project
now or will I have to resubmit ?

On Thu, 17 Feb 2022, 2:52 am Mateo Villagomez, ***@***.***>
wrote:

> Don't worry take your time 🤓
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tasnimulhasan07/skel-4/pull/1#issuecomment-1042515959>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AU5UHQUMG6WNARFSWIMBRJDU3RPFRANCNFSM5OOFJAUQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mateo951 reviewed (approved):
### Hi @tasnimulhasan07, 

Great job implementing all the required changes requested by your previous reviewers ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
